k8s.go

```diff
@@ -0,0 +1,247 @@
+package profitbricks
+
+import "net/http"
+
+type KubernetesClusters struct {
+	// URL to the collection representation (absolute path)
+	// Read Only: true
+	// Format: uri
+	Href string `json:"href,omitempty"`
+
+	// Unique representation for Kubernetes Cluster as a collection on a resource.
+	// Read Only: true
+	ID string `json:"id,omitempty"`
```

Comment:
Would your `// Enum: [k8s]` comment make sense here?

Reply:
I do not understand your comment.